GoogleSignIn/Tests/Unit/GIDEMMSupportTest.m

```diff
@@ -57,6 +63,35 @@ @interface GIDEMMSupportTest : XCTestCase
 
 @implementation GIDEMMSupportTest
 
+- (void)testEMMSupportDelegate {
+  XCTestExpectation *emmErrorExpectation = [self expectationWithDescription:@"EMM AppAuth error"];
+
+  GIDFailingOIDAuthState *authState = [GIDFailingOIDAuthState testInstance];
+  GIDGoogleUser *user = [[GIDGoogleUser alloc] initWithAuthState:authState profileData:nil];
+  GIDFakeFetcherService *fakeFetcherService = [[GIDFakeFetcherService alloc]
+                                                initWithAuthorizer:user.fetcherAuthorizer];
+
+  NSURLRequest *request = [NSURLRequest requestWithURL:[NSURL URLWithString:@""]];
+  GTMSessionFetcher *fakeFetcher = [fakeFetcherService fetcherWithRequest:request];
+
+  GIDTestWorker *testWorker = [[GIDTestWorker alloc] initWithFetcher:fakeFetcher];
```

Comment:
Thanks for updating this test. However, I still feel the `GIDTestWorker` isn't necessary -- it actually makes it a bit more difficult to read the code because the reader needs to jump between this file and GIDTestWorker.m to understand this test case.

One benefit of having `GIDTestWorker` is that it makes it clear that the "work" is going to fail by naming the method `failWorkWithCompletion:`. I think this is already implied by initializing `GIDGoogleUser` with a `GIDFailingOIDAuthState`. If you want to make this more explicit, you can add a comment right before calling `[fakeFetcher failWorkWithCompletion:]`.

Reply:
Resolving since I've removed `GIDTestWorker` and have renamed some things in the test to make the intention more clear.